pandas/tests/test_expressions.py

```diff
@@ -442,3 +442,19 @@ def test_bool_ops_warn_on_arithmetic(self):
                     r = f(df, True)
                     e = fe(df, True)
                     tm.assert_frame_equal(r, e)
+
+    def test_bool_ops_column_name_dtype(self):
```

Comment:
can you parameterize on both eq and ne here

Reply:
@jreback thank you your advice~! i can not understand how to parameterize eq and ne and why do parameterize... sorry i am not good at undertstanding english... thank you!

Reply:
@baidoosik : This is what @jreback is talking about:

https://docs.pytest.org/en/latest/parametrize.html

Perhaps that will help clarify?

Reply:
@pytest.mark.parametrize("index,has_tz", [
        (pd.date_range('2015-01-01 10:00', freq='D', periods=3,
                       tz='US/Eastern'), True),  # datetimetz
        (pd.timedelta_range('1 days', freq='D', periods=3), False),  # td
        (pd.period_range('2015-01-01', freq='D', periods=3), False)  # period
    ])

you recommend like above code style, is it right?

Reply:
Yes!  That's right.